config/logging.py

```diff
@@ -0,0 +1,57 @@
+logging_config = {
+  'version': 1,
+  'disable_existing_loggers': False,
+  "loggers": {
+    '': {  # root logger
+      'handlers': ['console', 'file'],
+      'level': 'INFO',
+      'propagate': True
+    },
+    'alembic.runtime.migration': {
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+    'urllib3.connectionpool': {
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+    'elasticsearch': {
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+    'ElasticIndex': {
+      'handlers': ['console', 'file'],
+      'level': 'WARN',
+      'propagate': False
+    },
+  },
+  "formatters": {
+    "simple": {
+      "format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    }
+  },
+  "root": {
+    "level": "DEBUG",
```

Comment:
Debug might me WAY too loud and create a bunch of junk.

Reply:
I set console to DEBUG and file to INFO